src/other/broadcasting.jl

```diff
@@ -0,0 +1,94 @@
+struct DummyColumnValue end
+
+Base.broadcastable(df::AbstractDataFrame) = Matrix(df)
+
+Base.maybeview(df::AbstractDataFrame, idxs...) = view(df, idxs...)
+
+function Base.maybeview(df::AbstractDataFrame, idxs)
+    if idxs isa Symbol
+        if !haskey(df, idxs)
+            if !(df isa DataFrame)
+                throw(ArgumentError("column $idxs not found"))
+            end
+            df[idxs] = Vector{DummyColumnValue}(undef, nrow(df))
+            return view(df, [idxs])
+        end
+    end
+    if idxs isa AbstractVector{Symbol}
+        for idx in idxs
+            if !haskey(df, idx)
+                if !(df isa DataFrame)
+                    throw(ArgumentError("column $idxs not found"))
+                end
+                df[idx] = Vector{DummyColumnValue}(undef, nrow(df))
+            end
+        end
+    end
+    view(df, idxs)
+end
+
+function Base.copyto!(df::SubDataFrame, bc)
+    m = Base.materialize(bc)
```

Comment:
I don't think you should need this `materialize` here.

Reply:
I materialized to avoid possible problems with aliasing (I might be modifying RHS which might appear later in processing in LHS).

Additionally in general if I do not `materialize` then what is the recommended way to access values stored in `bc`?